File: app/src/main/java/ng/com/techdepo/esidemtest/models/Question.java
package ng.com.techdepo.esidemtest.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Question implements Serializable {

    @SerializedName("id")
    int id;
    @SerializedName("question")
    String question;
    @SerializedName("option")
    Options options;
    @SerializedName("section")
    String section;
    @SerializedName("image")
    String image;
    @SerializedName("answer")
    String answer;
    @SerializedName("solution")
    String solution;
    @SerializedName("examtype")
    String examtype;
    @SerializedName("examyear")
    String examyear;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getQuestion() {
        return question;
    }

    public void setQuestion(String question) {
        this.question = question;
    }

    public Options getOptions() {
        return options;
    }

    public void setOptions(Options options) {
        this.options = options;
    }

    public String getSection() {
        return section;
    }

    public void setSection(String section) {
        this.section = section;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getAnswer() {
        return answer;
    }

    public void setAnswer(String answer) {
        this.answer = answer;
    }

    public String getSolution() {
        return solution;
    }

    public void setSolution(String solution) {
        this.solution = solution;
    }

    public String getExamtype() {
        return examtype;
    }

    public void setExamtype(String examtype) {
        this.examtype = examtype;
    }

    public String getExamyear() {
        return examyear;
    }

    public void setExamyear(String examyear) {
        this.examyear = examyear;
    }
}

File: app/src/main/java/ng/com/techdepo/esidemtest/models/Question.kt
package ng.com.techdepo.esidemtest.models

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class Question : Serializable {

    @SerializedName("id")
    var id: Int? = 0
    @SerializedName("question")
    var question: String?=null
    @SerializedName("option")
    var options: Options?=null
    @SerializedName("section")
    var section: String?=null
    @SerializedName("image")
    var image: String?=null
    @SerializedName("answer")
    var answer: String?=null
    @SerializedName("solution")
    var solution: String?=null
    @SerializedName("examtype")
    var examtype: String?=null
    @SerializedName("examyear")
    var examyear: String?=null

    constructor(id: Int, question: String, section: String, options: Options, answer: String, examtype: String, examyear: String) {
        this.id = id
        this.question = question
        this.options = options
        this.answer = answer
        this.examtype = examtype
        this.examyear = examyear
        this.section = section
    }

    constructor()
}
